src/ngraph_builder.cc

```diff
@@ -633,8 +633,152 @@ tf::Status Builder::TranslateGraph(const std::vector<tf::TensorShape>& inputs,
 
       ng_op_map[op->name()] = ng_conv;
     }
+    // ------
+    // Conv2DBackpropInput
+    // ------
+    else if (op->type_string() == "Conv2DBackpropInput") {
+      if (op->num_inputs() != 3) {
+        return tf::errors::InvalidArgument(
+            "Number of inputs is not 3 for Conv2DBackpropInput");
+      }
+
+      tf::Node *tf_filter, *tf_out_backprop;
+      TF_RETURN_IF_ERROR(op->input_node(1, &tf_filter));
+      TF_RETURN_IF_ERROR(op->input_node(2, &tf_out_backprop));
+      auto ng_filter = ng_op_map.at(tf_filter->name());
```

Comment:
Need to catch the `std::out_of_range` exception if the lookup fails. See this: https://en.cppreference.com/w/cpp/container/map/at

Reply:
You'd think I would know about this, having done it before :|